### PR TITLE
Add nord color scheme

### PR DESCRIPTION
**Pull request by marcplustwo**
I use the [nord color scheme](https://github.com/arcticicestudio/nord) on my machine and adapted it for this project.

### CODE REVIEW

**Comment by PickingUpPieces** [comment]
Would be great to have more colour variations :D @bobthecow

**Comment by marcplustwo** [comment]
@PickingUpPieces Any specific requests or ideas?

I noticed that there are still some issues e.g. the color for the background process indicator doesn't quite fit. Will fix.

File: functions/__bobthefish_colors.fish [hunk 1]
```diff
@@ -480,6 +480,54 @@ function __bobthefish_colors -S -a color_scheme -d 'Define colors used by bobthe
       set -x color_virtualgo             $cyan $bg --bold
       set -x color_desk                  $comment $bg --bold
 
+    case 'nord'
+      set -l base00  2E3440
+	  set -l base01  3B4252
```
Comment (bobthecow) [inline functions/__bobthefish_colors.fish]: it looks like there's some whitespace weirdness going on here.

**Comment by marcplustwo** [comment]
I added the screenshot to the README and fixed the whitespace/mixed-indents.

**Pull request merged**